private/bufpkg/bufcheck/internal/helper.go

```diff
@@ -110,10 +110,17 @@ func (h *Helper) addFileAnnotationf(
 	format string,
 	args ...interface{},
 ) {
+	baseLocs := []bufprotosource.Location{location}
```

Comment:
nit: would probably name this `ignoreLocations` or just `locations` (since it is being passed to the `ignoreFunc` later anyway). I would probably also adjust the subsequent variables, `%s/locDescriptor/locationDescriptor` and `%s/descLocation/descriptorLocation` to adhere to the fuller naming conventions.

Reply:
done